chirp/drivers/baofeng_uv17Pro.py

```diff
@@ -742,6 +777,163 @@ def apply_Key2short(setting, obj):
                           RadioSettingValueBoolean(_mem.settings.fmenable))
         basic.append(rs)
 
+    def get_settings_common_workmode(self, workmode, _mem):
+
+        vfoA = RadioSettingGroup("vfoA", "VFO A")
+        vfoB = RadioSettingGroup("vfoB", "VFO B")
+
+        def convert_bytes_to_freq(bytes):
```

Comment:
This gets copied as a closure into each driver that needs this functionality, so we have this duplicated in a number of places. We really need to put this in a single place (maybe `baofeng_common.py`?) instead of doing this. Fancy doing that refactor for karma points? :)

Reply:
Happy to! I'll take a look on a slow day if nobody beats me to it